lib/oracle.js

```diff
@@ -7,8 +7,14 @@ import {validateQueryPayload} from "./validate.js";
 
 const READ_ONLY = new Set(["SELECT", "USAGE", "CONNECT"]);
 
+const boolean = ["null", "boolean"],
```

Comment:
Can this be moved into the scope of the constructor? It seems like a potential source of confusion having variables with the same names as primitive types at the module scope.